nnfabrik/templates/trained_model.py

```diff
@@ -26,6 +26,33 @@ class TrainedModelBase(dj.Computed):
         of `Fabrikant`, `Seed`, `Dataset`, `Model`, and `Trainer`, respectively. You could also set the
         value of `nnfabrik` to a module or "core" as stated above, and specifically override a target table
         via setting one of the table class property as well.
+    * The TrainedModel table also needs a storage defined within the dj.config.
+        By default, this storage is called "minio", and is set by this template accordingly.
```

Comment:
I like the explanation for setting up the storage, but I feel this ought to belong not in doc for TrainedModelBase but preferably somewhere closer to the doc's top level with its section. You can then refer to it from here.